File: APIconvenios/DTOs/Empresa/EmpresaWithConveniosDto.cs
using APIconvenios.DTOs.ConvenioEspecifico;
using APIconvenios.DTOs.ConvenioMarco;
using APIconvenios.Models;

namespace APIconvenios.DTOs.Empresa
{
    public class EmpresaWithConveniosDto
    {
        public string NombreEmpresa { get; set; }   
        public ConvenioMarcoDto? ConvenioMarco { get; set; }
        public List<ConvenioEspecificoDto> conveniosEspecificos { get; set; }  
    }
}

File: UserInterface/src/Types/Empresa/EmpresaWithConveniosDto.ts
import type { ConvenioMarcoDto } from '@/Types/ViewModels/ViewModels'
import type { ConvenioEspecificoDto } from '@/Types/ViewModels/ViewModels'

export interface EmpresaWithConveniosDto {
  idEmpresa: number
  nombreEmpresa: string
  convenioMarco: ConvenioMarcoDto | null
  conveniosEspecificos: ConvenioEspecificoDto[]
}
